pkg/cluster/gitops.go

```diff
@@ -15,29 +15,16 @@ func gitOpsEntry() *ConfigManagerEntry {
 		},
 		Processors: []ParsedProcessor{processGitOps},
 		Defaulters: []Defaulter{
-			func(c *Config) error {
-				if c.GitOpsConfig != nil {
-					c.GitOpsConfig.SetDefaults()
-				}
-				return nil
-			},
+			setGitOpsDefaults,
 			SetDefaultFluxGitHubConfigPath,
+			setFluxDefaults,
+			SetDefaultFluxConfigPath,
```

Comment:
nit: we don't need to register the defaults for flux here, those will be already added by the flux entry

Reply:
ah ok I see, because the FluxConfig entry here https://github.com/aws/eks-anywhere/blob/c5e281d6a1a008fc4c0efb095d22a29e4e402a3d/pkg/cluster/default_manager.go#L12

is processed after the GitOps entry, we'll have the FluxConfig present in the spec and it'll get defaulted and vaildated

ok thanks!

Reply:
https://github.com/aws/eks-anywhere/pull/1827/commits/e0c1cea03b3576d9edf4a990155eec4410d6222d